### PR TITLE
feat: Set up Circle CI

### LINKED ISSUE #3: Setup circleci
Circleci should build the Dockerfile and deploy it to docker hub.

We'll also need to coordinate w/ Operations/SRE to:

- setup a new docker hub project it'll deploy to
- add the dockerhub credentials (as a secret) to this circleci project

### PR DESCRIPTION
**Pull request by ethowitz**
## Description

Adds a Circle CI job to run flake8 on `loadtest.py`.

## Testing

1. Install the Circle CI local tool, following the instructions found [here](https://circleci.com/docs/2.0/local-cli/#installation)
2. Create a `process.yml` file with `circleci config process .circleci/config.yml > process.yml`
3. Run the `test` job locally with `circleci local execute -c process.yml --job test`

## Issue(s)

Closes #3

### CODE REVIEW

**Comment by ethowitz** [comment]
I will rebase this against `main` when #4 and #5 are merged

**Comment by ethowitz** [comment]
I'll coordinate with ops to add `DOCKER_USER` and `DOCKER_PASS` to this CircleCI project before I merge to main

**Comment by ethowitz** [comment]
Also, for some reason I am being given the option to merge even though there has not been an approving review 🤔

**Review by jrconlin — DISMISSED** [review]
I think that there's a few branch management settings you can switch on to ensure that even code owners have to get approval before merging.
Granted, you can go in later and turn them off, so it's more a formality than anything.

**Comment by ethowitz** [comment]
I realized the reason I was given the option to merge was because I still had the base branch set to feat/2-set-up-docker (once I switched it to main, it wouldn't let me click merge). I was hoping no one would see my deleted comment about that 😛

**Comment by jrconlin** [comment]
Heh, github is always "improving" things, so I completely expected there to be a new set of options. ;)

File: .circleci/config.yml
Comment (jrconlin) [inline .circleci/config.yml:9]: Good to have, but I'm not sure we've got anything that returns this. Normally it's part of `/__version__`

**Comment by ethowitz** [comment]
Closing as we're stopping work on these load tests

**Comment by jbuck** [comment]
Circle CI env vars have been configured for Docker Hub

**Review by jrconlin — APPROVED** [review]

File: .circleci/config.yml
Comment (jbuck) [reply .circleci/config.yml:9]: Yes, please add this to the version endpoint!

Comment (jrconlin) [reply .circleci/config.yml:9]: Right, but since this is a set of scripts that are used to run testing against the contile service, there's no `/__version__` endpoint to speak of. These should not be set up as a normal service (with the other dockerflow endpoints), but instead be used as a base for molotov or similar test runs.

**Review by pjenvey — APPROVED** [review]

**Pull request merged**
